pinot-segment-local/src/main/java/org/apache/pinot/segment/local/segment/index/text/TextIndexConfigBuilder.java

```diff
@@ -106,6 +106,11 @@ public TextIndexConfig.AbstractBuilder withProperties(@Nullable Map<String, Stri
           _fstType = FSTType.LUCENE;
         }
       }
+
+      if (textIndexProperties.get(FieldConfig.TEXT_INDEX_LUCENE_CASE_SENSITIVE_INDEX) != null) {
```

Comment:
Most of the other configs (e.g. `stopWordsInclude`) are using the new json format to config. We probably don't need to add support for old properties config

Reply:
done

Reply:
What I meant is that we don't need to introduce `FieldConfig.TEXT_INDEX_LUCENE_CASE_SENSITIVE_INDEX`. You may modify the test table config to use the json format of the config